packages/lib/src/components/Badge/TaskStatusBadge/index.tsx

```diff
@@ -35,24 +35,18 @@ const TaskStatusBadge: React.FC<TaskStatusBadgeProps> = ({ status }) => {
 
   const statusText = taskStatusToCopy[status] ?? status;
 
-  if (status in mapTaskStatusToBadgeVariant) {
-    return (
-      <StyledBadge
-        key={status}
-        css={badgeWidthMaxContent}
-        data-cy="task-status-badge"
-        variant={mapTaskStatusToBadgeVariant[status]}
-      >
-        {statusText}
-      </StyledBadge>
-    );
-  }
-
   return (
     <StyledBadge
       key={status}
+      css={badgeWidthMaxContent}
```

Comment:
is it possible `badgeWidthMaxContent` is redundant if it's already included in `StyledBadge`?

Reply:
Totally was a copy pasta from merging the separate implementations